fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -5,6 +5,7 @@
 from .moment import ClassificationMoment
 from .moment import _GROUP_ID, _LABEL, _PREDICTION, _ALL, _EVENT, _SIGN
 from .error_rate import ErrorRate
+import numpy as np
```

Comment:
Is this for `dropna()` ?

Reply:
No, this can be removed. This was originally added because of some NumPy array manipulation that I was trying for the DataFrame, which I am not using now.